cmd/target-browser/cmd/metadata.go

```diff
@@ -0,0 +1,26 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// metadataCmd represents the metadata command
+var metadataCmd = &cobra.Command{
+	Use:   metadataCmdName,
+	Short: "A brief description of your command",
```

Comment:
fix description.

Reply:
will be fixed .. when @pankaj9310's PR changes are integrated. This is just mock up.